console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx

```diff
@@ -20,7 +20,6 @@ import { StyleSheet, css } from "aphrodite";
 import {
   StatusTypes,
   ProjectStatus,
-  ProjectError,
```

Comment:
@suyash-naithani I already raised a PR https://github.com/EnMasseProject/enmasse/pull/5075 to comment out the error count from  the project list. Could you please undo these changes?

Reply:
@drcoolsanjeev Okay, will do

Reply:
@drcoolsanjeev Removed the error count conflict. This PR is solely for updating  minor changes to ID's for the project